test/functional/p2p_filter.py

```diff
@@ -34,6 +37,13 @@ class FilterNode(P2PInterface):
         nFlags=1,
     )
 
+    def __init__(self, fRelay=True):
+        super().__init__()
+        if fRelay == False:
+            version_without_nRelay = msg_version()
+            version_without_nRelay.nRelay = 0
+            self.on_connection_send_msg = version_without_nRelay
```

Comment:
How does this work? Isn't `peer_connect` called *after* init, in which case `on_connection_send_msg` would be overwritten?

Reply:
Aw shoot, just fixed using `send_version=False` and then manually sending it after connection is established. Asserting `relaytxes=False` so it should be set correctly now.